dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/RequestItemConverter.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.rest.converter;
+
+import java.sql.SQLException;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.servlet.http.HttpServletRequest;
+
+import org.dspace.app.requestitem.RequestItem;
+import org.dspace.app.requestitem.service.RequestItemService;
+import org.dspace.app.rest.model.RequestItemRest;
+import org.dspace.app.rest.utils.ContextUtil;
+import org.dspace.core.Context;
+import org.dspace.services.RequestService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Convert between {@link org.dspace.app.requestitem.RequestItem} and
+ * {@link org.dspace.app.rest.model.RequestItemRest}.
+ *
+ * @author Mark H. Wood <mwood@iupui.edu>
+ */
+@Named
+public class RequestItemConverter
+        extends DSpaceConverter<RequestItem, RequestItemRest> {
+    private static final Logger LOG = LoggerFactory.getLogger(RequestItemConverter.class);
+
+    @Inject
+    protected BitstreamConverter bitstreamConverter;
+
+    @Inject
+    protected ItemConverter itemConverter;
+
+    @Inject
+    protected RequestItemService requestItemService;
+
+    @Inject
+    protected RequestService requestService;
+
+    @Override
+    public RequestItemRest fromModel(RequestItem requestItem) {
+        RequestItemRest requestItemRest = new RequestItemRest();
+        requestItemRest.setAccept_request(requestItem.isAccept_request());
+        requestItemRest.setAllfiles(requestItem.isAllfiles());
+        requestItemRest.setBitstream(bitstreamConverter.fromModel(requestItem.getBitstream()));
+        requestItemRest.setDecision_date(requestItem.getDecision_date());
+        requestItemRest.setExpires(requestItem.getExpires());
+        requestItemRest.setId(requestItem.getID());
+        requestItemRest.setItem(itemConverter.fromModel(requestItem.getItem()));
+        requestItemRest.setReq_email(requestItem.getReqEmail());
```

Comment:
personally I prefer the camel case approach that is widely used in the rest of the code. Anyway, it is better than the current "JAVA" request item class that mix both approach